[16.0][MIG] sale_timesheet_rounded: Migration to 16.0

hoangphuc1410 opened the pull request:
In the version 16.0, we have function _timesheet_preprocess is always update 'project_id' in model 'account.analytic.line'. So we don't have case no project_id in function 'test_analytic_line_read_override'. So I was delete line with project_id = False.
I remove WARNING in README because force_compute was removed from version 13.0 .

Commit:
[ADD] crm_timesheet

chienandalu committed:
[MIG] crm_timesheet: Migration 10.0

pedrobaeza committed:
[IMP] crm_timesheet: Better layout

pedrobaeza committed:
[IMP] crm_timesheet: Handle better project/analytic account duality

yajo committed:
[FIX][crm_timesheet] Do not restrict project

This addon assumed one will always fill the analytic account before the project, but current behavior does not let you choose a project outside current analytic account, which in practice means you can never change the project if `analytic_account_id` field is hidden (as it happens in other addons such as `project_issue_timesheet_time_control`).

Now, if you choose a project it will fill the analytic account, and also the other way around, so why bother about the domains?

pedrobaeza committed:
[IMP] crm_timesheet: Use projects

As now hr_timesheet bases all their functionality on projects,
there's no sense to keep the old behavior linking crm.lead times
to analytic account instead of projects, so I have changed this,
making easier the associated logic (which, in the other hand, was
not fully correct previously).

benwillig committed:
[ADD] project_task_stage_allow_timesheet: allow timesheets on a task depending on its stage

MiquelRForgeFlow committed:
[11.0][MIG] hr_timesheet_sheet (#125)

* [10.0] hr_timesheet_sheet

* [11.0][MIG] hr_timesheet_sheet

* [REMOVE] hr_timesheet.sheet.account

* [REMOVE] 'new' state

* [ADD] Tests

* [UPD] Adapt to multicompany

* [ADD] Add more tests (include multicompany tests)

* [FIX] project_task_stage_allow_timesheet: show error message only if task

* [ADD] Migration scripts to v11

cubells committed:
[MIG] crm_timesheet: Migration to 11.0

alexey-pelykh committed:
[ADD] hr_timesheet_report: generate configurable Timesheet Reports from Task Logs

[UPD] Update hr_timesheet_report.pot

[UPD] README.rst

[ADD] icon.png

pedrobaeza committed:
[IMP] crm_timesheet: Improve views:

* Limit timesheet field addition in view to authorized users
* Use proper view in lead embedded timesheet page
* Don't show lead field in embedded timesheet page

Commit:
Added translation using Weblate (Spanish)

Commit:
[12.0] crm_timesheet: Migration to 12.0

ddufresne committed:
[12.0] Migrate project_task_stage_allow_timesheet

alexey-pelykh committed:
[ADD] sale_timesheet_task_exclude: exclude specific task from Sale Order

alexey-pelykh committed:
[ADD] sale_timesheet_line_exclude: exclude Timesheet Line from Sale Order

alexey-pelykh committed:
[12.0][FIX] Manifest fixes (#255)

* [FIX] sale_timesheet_line_exclude: fix manifest
* [FIX] sale_timesheet_task_exclude: fix manifest

alexey-pelykh committed:
[12.0][FIX] Manifest fixes (#255)

* [FIX] sale_timesheet_line_exclude: fix manifest
* [FIX] sale_timesheet_task_exclude: fix manifest

alexey-pelykh committed:
[FIX] project_task_stage_allow_timesheet: check if stage is set

simahawk committed:
Add sale_timesheet_rounded

sale_timesheet_rounded: fix do not use default

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

marylla committed:
Added translation using Weblate (German)

yajo committed:
[IMP] crm_timesheet: ❤️ 4 UX

Following the changes from https://github.com/OCA/project/pull/586, here I implement the same UX when using it with CRM leads.

yajo committed:
[IMP] crm_timesheet: Tell the user no running line was found

Imagine this scenario:

1. In tab 1 of the browser, you have opened lead 1.
2. In tab 2 of the browser, you have opened lead 2.
3. You go to tab 1 and start a timer.
4. Work, work, work...
5. You go to tab 2 and start a timer, stopping that of lead 1.
6. Work, work, work...
7. You go to tab 1 and see that timer as running (it is not, but you didn't refresh). You hit stop.

Before this commit, it just seemed like the timer was actually stopped. What did happen behind the scenes is that your view was refreshed, but no timer was touched fortunately.

After this commit, you get a message telling you that there's no timer to stop and that your browser is most likely out of sync. This mimics the behavior previously found when doing the same, but directly in the AAL. Now it's present in leads too.

yajo committed:
[IMP] project_timesheet_time_control: Refactor with mixin

Models related to timesheet time controls now inherit from a mixin that adds most needed logic automatically.

This requires the changes introduced in https://github.com/OCA/project/pull/596.

alexey-pelykh committed:
[ADD] hr_timesheet_sheet: policies

astirpe committed:
[FIX] hr_timesheet_sheet_policy*: admin always reviewer

alexey-pelykh committed:
hr_timesheet_sheet_autodraft: automatically create sheets

Commit:
Added translation using Weblate (Spanish)

OCA-git-bot committed:
Update hr_timesheet_sheet_autodraft.pot

[UPD] README.rst

[ADD] icon.png

josepmy committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: timesheet-12.0/timesheet-12.0-project_task_stage_allow_timesheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-project_task_stage_allow_timesheet/es/

josepmy committed:
Added translation using Weblate (Spanish)

josepmy committed:
Added translation using Weblate (Spanish)

josepmy committed:
Translated using Weblate (Spanish)

Currently translated at 7.7% (1 of 13 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet_policy_project_manager
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet_policy_project_manager/es/

Translated using Weblate (Spanish)

Currently translated at 100.0% (13 of 13 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet_policy_project_manager
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet_policy_project_manager/es/

Translated using Weblate (Spanish)

Currently translated at 100.0% (13 of 13 strings)

Translation: timesheet-12.0/timesheet-12.0-hr_timesheet_sheet_policy_project_manager
Translate-URL: https://translation.odoo-community.org/projects/timesheet-12-0/timesheet-12-0-hr_timesheet_sheet_policy_project_manager/es/

Commit:
[IMP] sale_timesheet_rounded: black, isort, prettier

Commit:
[MIG] sale_timesheet_rounded: Migration to 13.0

Chandresh-SerpentCS committed:
[ADD] Ability to add time type in timesheet lines.

pedrobaeza committed:
[IMP] sale_timesheet_rounded: Convert it to computed writeable

This way, we avoid a lot of code and get the same features.

dreispt committed:
[REF] hr_timesheet_time_type: remove redundant dependency

enriquemartin committed:
12.0 - sale_timesheet_rounded

fixes issue #687

davejames committed:
[IMP] hr_timesheet_sheet_policy_project_manager: black, isort, prettier

davejames committed:
[MIG] hr_timesheet_sheet_policy_project_manager: Migration to 13.0

davejames committed:
hr_timesheet_sheet_autodraft: black, isort, prettier

[IMP] hr_timesheet_sheet_autodraft: black, isort, prettier

davejames committed:
hr_timesheet_sheet_autodraft: Migration to 13.0

oca-travis committed:
Update hr_timesheet_sheet_autodraft.pot

OCA-git-bot committed:
README.rst

flachica committed:
[IMP] sale_timesheet_task_exclude: black, isort

flachica committed:
[13.0] sale_timesheet_task_exclude: Migration to 13.0

pedrobaeza committed:
[IMP] crm_timesheet: black, isort, prettier

pedrobaeza committed:
[MIG] crm_timesheet: Migration to 13.0

- README and manifest adapted
- Views changed

alexey-pelykh committed:
[UPD] Brainbean Apps => CorporateHub

alexey-pelykh committed:
[UPD] Brainbean Apps => CorporateHub

[UPD] README.rst

alexey-pelykh committed:
[UPD] Brainbean Apps => CorporateHub

alexey-pelykh committed:
Brainbean Apps => CorporateHub

alexey-pelykh committed:
[UPD] Brainbean Apps => CorporateHub

alexey-pelykh committed:
[UPD] Brainbean Apps => CorporateHub

OCA-git-bot committed:
README.rst

dreispt committed:
[FIX] sale_timesheet_line_exclude: user friendly Non-billable label

Shivams7 committed:
[MIG] Migrated hr_timesheet_time_type module from v12 to v13.

ypapouin committed:
Added translation using Weblate (French)

dsolanki-initos committed:
hr_timesheet_sheet_autodraft: isort, black, prettier.

dsolanki-initos committed:
Migrate module hr_timesheet_sheet_autodraft to v14.

fshah-initos committed:
[IMP] hr_timesheet_sheet_policy_project_manager: Black, isort, prettier.

fshah-initos committed:
[MIG] Migrate module "hr_timesheet_sheet_policy_project_manager" to v14.

dsolanki-initos committed:
[IMP] sale_timesheet_task_exclude: Black, Isort, Prettier.

fshah-initos committed:
[IMP] sale_timesheet_rounded: black, isort, prettier

fshah-initos committed:
[14.0][MIG]sale_timesheet_rounded

dsolanki-initos committed:
[MIG] Migrate module sale_timesheet_task_exclude to v14.

fshah-initos committed:
[IMP] crm_timesheet: Black, isort, prettier.

hkapatel-initos committed:
[MIG] Migrate module crm_timesheet to v14.

AmmarOfficewalaSerpentCS committed:
[FIX] Fixed travis.

AmmarOfficewalaSerpentCS committed:
[14.0][MIG] Migrated hr_timesheet_time_type module.

oca-travis committed:
Update hr_timesheet_sheet_autodraft.pot

OCA-git-bot committed:
README.rst

Du-ma committed:
[IMP] sale_timesheet_line_exclude: black, isort, prettier

Du-ma committed:
[MIG] sale_timesheet_line_exclude: migration to 13.0

max3903 committed:
Apply suggestions from code review

dsolanki-initos committed:
[IMP] hr_timesheet_report: black, isort, prettier

dsolanki-initos committed:
[MIG] Migrate module hr_timesheet_report to v14.

[UPD] Update hr_timesheet_report.pot

[UPD] README.rst

bosd committed:
Added translation using Weblate (Dutch)

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (12 of 12 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet_autodraft
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet_autodraft/nl/

OCA-git-bot committed:
sale_timesheet_rounded 14.0.1.0.1

OCA-git-bot committed:
update dotfiles [ci skip]

Commit:
[FIX] sale_timesheet_round: test need an install decorator

dantiller2 committed:
[MIG] sale_timesheet_line_exclude: Migration to 14.0

Commit:
[15.0] MIG sale_timesheet_rounding: Migration to 15.0

Commit:
[15.0] MIG sale_timesheet_rounding: Fixing precommit error

francesco-ooops committed:
Added translation using Weblate (Italian)

Translated using Weblate (Italian)

Currently translated at 100.0% (78 of 78 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_report
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_report/it/

Freni-OSI committed:
[15.0][MIG] sale_timesheet_line_exclude

Freni-OSI committed:
[15.0][MIG] sale_timesheet_task_exclude

Murtaza-SerpentCS committed:
[15.0][MIG] hr_timesheet_time_type

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

Translated using Weblate (Portuguese)

Currently translated at 98.7% (77 of 78 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_report
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_report/pt/

Yvesldff committed:
Added translation using Weblate (French)

Translated using Weblate (French)

Currently translated at 92.3% (72 of 78 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_report
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_report/fr/

victoralmau committed:
[ADD] hr_timesheet_employee_analytic_tag: New addon

TT39712

[UPD] Update hr_timesheet_employee_analytic_tag.pot

[UPD] README.rst

vincent-hatakeyama committed:
Added translation using Weblate (French)

vincent-hatakeyama committed:
Translated using Weblate (French)

Currently translated at 100.0% (5 of 5 strings)

Translation: timesheet-15.0/timesheet-15.0-sale_timesheet_line_exclude
Translate-URL: https://translation.odoo-community.org/projects/timesheet-15-0/timesheet-15-0-sale_timesheet_line_exclude/fr/

sergiocorato committed:
[13.0][FIX] sale_timesheet_task_exclude self is a recordset

Commit:
[MIG] crm_timesheet: Migration to 15.0

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_employee_analytic_tag
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_employee_analytic_tag/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (12 of 12 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet_autodraft
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet_autodraft/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (78 of 78 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_report
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_report/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (78 of 78 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_report
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_report/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (78 of 78 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_report
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_report/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (78 of 78 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_report
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_report/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (78 of 78 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_report
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_report/it/

[UPD] README.rst

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_report
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_report/

Translated using Weblate (Italian)

Currently translated at 100.0% (78 of 78 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_report
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_report/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (12 of 12 strings)

Translation: timesheet-14.0/timesheet-14.0-hr_timesheet_sheet_autodraft
Translate-URL: https://translation.odoo-community.org/projects/timesheet-14-0/timesheet-14-0-hr_timesheet_sheet_autodraft/it/

mymage committed:
Added translation using Weblate (Italian)

Commit:
[IMP] sale_timesheet_line_exclude: black, isort, prettier

Commit:
[MIG] sale_timesheet_line_exclude: Migration to 16.0

bizzappdev committed:
hr_timesheet_sheet_autodraft: black, isort, prettier

bizzappdev committed:
hr_timesheet_sheet_autodraft: Migration to 15.0

FrancoMaxime committed:
[IMP] project_task_stage_allow_timesheet - pre-commit stuff

FrancoMaxime committed:
[16.0][MIG] project_task_stage_allow_timesheet: Migration to 16.0

Commit:
[15.0][FIX] sale_timesheet_rounded: avoid recomputing rounded amounts when posting an invoice

Yadier-Tecnativa committed:
[IMP] hr_timesheet_sheet_policy_project_manager: pre-commit stuff

Commit:
[IMP] sale_timesheet_task_exclude: black, isort, prettier

Commit:
[MIG] sale_timesheet_task_exclude: Migration to 16.0

hoangphuc1410 committed:
[16.0][MIG] sale_timesheet_rounded: Migration to 16.0

dzungtran89 commented:
Thanks, I got two issues:
- The test is failed `psycopg2.errors.NotNullViolation: null value in column "plan_id" of relation "account_analytic_account" violates not-null constraint
DETAIL:  Failing row contains (51, null, null, null, 7, null, 26, 26, AVG account, null, t, 2023-07-20 09:50:42.319817, 2023-07-20 09:50:42.319817).
`
- Please use [oca-gen-addon-readme](https://github.com/OCA/maintainer-tools#readme-generator) to update the readme

dzungtran89 reviewed (approved):
lgtm

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

OCA-git-bot committed:
Merge PR #569 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update sale_timesheet_line_exclude.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

Yadier-Tecnativa committed:
[MIG] hr_timesheet_sheet_policy_project_manager: Migration to 16.0

OCA-git-bot committed:
Merge PR #598 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update hr_timesheet_sheet_policy_project_manager.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet_policy_project_manager
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet_policy_project_manager/

micheledic committed:
[MIG] hr_timesheet_sheet_autodraft: Migration to 16.0

victoralmau committed:
[IMP] hr_timesheet_employee_analytic_tag: pre-commit stuff

victoralmau committed:
[MIG] hr_timesheet_employee_analytic_tag: Migration to 16.0

TT42244

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (183 of 183 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (10 of 10 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_task_required
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_task_required/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (9 of 9 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_task_stage
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_task_stage/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_task_domain
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_task_domain/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_name_customer
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_name_customer/es/

manuelregidor committed:
[IMP] hr_timesheet_time_type: black, isort, prettier

manuelregidor committed:
[MIG]hr_timesheet_time_type: Migration to 16.0

OCA-git-bot committed:
Merge PR #608 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update hr_timesheet_sheet.pot

Commit:
[UPD] Update hr_timesheet_time_type.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
Merge PR #607 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update hr_timesheet_employee_analytic_tag.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_employee_analytic_tag
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_employee_analytic_tag/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/es/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (16 of 16 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_time_type
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_time_type/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/es/

carolinafernandez-tecnativa committed:
[IMP] crm_timesheet: pre-commit stuff

carolinafernandez-tecnativa committed:
[MIG] crm_timesheet: Migration to 16.0

OCA-git-bot committed:
Merge PR #610 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update hr_timesheet_sheet.pot

Commit:
[UPD] Update crm_timesheet.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-sale_timesheet_line_exclude
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-sale_timesheet_line_exclude/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/

MiquelRForgeFlow committed:
[FIX] hr_timesheet_sheet: `image_128` -> `avatar_128`

OCA-git-bot committed:
Merge PR #613 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-crm_timesheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-crm_timesheet/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-crm_timesheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-crm_timesheet/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (10 of 10 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet_policy_project_manager
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet_policy_project_manager/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (182 of 182 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (182 of 182 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (16 of 16 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_time_type
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_time_type/es/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (182 of 182 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (11 of 11 strings)

Translation: timesheet-16.0/timesheet-16.0-crm_timesheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-crm_timesheet/it/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (11 of 11 strings)

Translation: timesheet-16.0/timesheet-16.0-crm_timesheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-crm_timesheet/es/

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (7 of 7 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_name_customer
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_name_customer/pt_BR/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (182 of 182 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/pt_BR/

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (10 of 10 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet_policy_project_manager
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet_policy_project_manager/pt_BR/

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (6 of 6 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_begin_end
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_begin_end/pt_BR/

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (9 of 9 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_task_stage
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_task_stage/pt_BR/

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_task_domain
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_task_domain/pt_BR/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (11 of 11 strings)

Translation: timesheet-16.0/timesheet-16.0-crm_timesheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-crm_timesheet/pt_BR/

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (3 of 3 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_employee_analytic_tag
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_employee_analytic_tag/pt_BR/

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (16 of 16 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_time_type
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_time_type/pt_BR/

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 40.0% (4 of 10 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_task_required
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_task_required/pt_BR/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (10 of 10 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_task_required
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_task_required/pt_BR/

adrianojprado committed:
Added translation using Weblate (Portuguese (Brazil))

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translation: timesheet-16.0/timesheet-16.0-sale_timesheet_line_exclude
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-sale_timesheet_line_exclude/pt_BR/

remi-filament committed:
Translated using Weblate (French)

Currently translated at 95.0% (173 of 182 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/fr/

remi-filament committed:
Added translation using Weblate (French)

remi-filament committed:
Translated using Weblate (French)

Currently translated at 100.0% (7 of 7 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_name_customer
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_name_customer/fr/

Commit:
set sale_timesheet_rounded as a rebel module (issue on sale_timesheet tests)

Commit:
[UPD] update copier v1.18

pedrobaeza committed:
Merge pull request #619 from c4a8-odoo/16.0-upd-copier

[UPD] update copier v1.18

Commit:
[UPD] Update hr_timesheet_sheet.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/

pere-aquarian committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/ca/

pere-aquarian committed:
Added translation using Weblate (Catalan)

pere-aquarian committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (6 of 6 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_begin_end
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_begin_end/ca/

OCA-git-bot committed:
Merge PR #600 into 16.0

Signed-off-by alexey-pelykh

Commit:
[UPD] Update sale_timesheet_task_exclude.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-sale_timesheet_task_exclude
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-sale_timesheet_task_exclude/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

OCA-git-bot committed:
Merge PR #621 into 16.0

Signed-off-by dreispt

Commit:
[UPD] Update project_task_stage_allow_timesheet.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (181 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/it/

Commit:
[ADD] sale_timesheet_analytic_propagation: update timesheet analytic account from task.

This module allows to update the timesheet's analytic account when the task related to
it changes the analytic account. This only happens if the timesheet is not invoiced yet.

@moduon MT-5142

OCA-git-bot committed:
Merge PR #638 into 16.0

Signed-off-by alexey-pelykh

Commit:
[UPD] Update project_task_analytic_propagation.pot

OCA-git-bot committed:
[BOT] post-merge updates

jakobkrabbe committed:
Translated using Weblate (Swedish)

Currently translated at 99.4% (180 of 181 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet/sv/

jakobkrabbe committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (9 of 9 strings)

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_task_stage
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_task_stage/sv/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: timesheet-16.0/timesheet-16.0-project_task_analytic_propagation
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-project_task_analytic_propagation/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: timesheet-16.0/timesheet-16.0-project_task_stage_allow_timesheet
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-project_task_stage_allow_timesheet/it/

victoralmau committed:
[IMP] hr_timesheet_report: pre-commit stuff

victoralmau committed:
[MIG] hr_timesheet_report: Migration to 16.0

TT47834

OCA-git-bot committed:
Merge PR #642 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update hr_timesheet_report.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #634 into 16.0

Signed-off-by alexey-pelykh

Commit:
[UPD] Update hr_timesheet_sheet_autodraft.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: timesheet-16.0/timesheet-16.0-hr_timesheet_sheet_autodraft
Translate-URL: https://translation.odoo-community.org/projects/timesheet-16-0/timesheet-16-0-hr_timesheet_sheet_autodraft/
